README.md

```diff
@@ -24,12 +24,6 @@ For a guide how to get started with this Docker image please take a look at our
 
 * 8080
 
-## Available environment variables
```

Comment:
This is intentionally added to make clear this docker image does not define specific env variables, it's structured like that for all docker image repos.